### DISCUSSION TITLE
Is there a boms file for the M27C322 adapter?

### DISCUSSION THREAD

**Discussion by hualazimo7** [sanni/cartreader#909] [Need help]
There is no silkscreen on the board.
Anyone has made this adapter?
Is it useful? I'd like to reflash M27c322 and replace the mask rom on the snes SA-1 cartridges.

![pic](https://github.com/sanni/cartreader/assets/74229525/17928167-814f-487f-961c-03e437727925)

**Comment by sanni** [comment]
The 27C322 code never worked correctly and I gave up eventually. The issue was that flashing the 27C322 often didn't work and even if it did the data would delete itself after a year or so.

Here is more information:
https://web.archive.org/web/20190722053104/https://github.com/sanni/cartreader/wiki/M27C322

To build it you need to buy a little 12V step-up converter on eBay two times and transfer the parts to the adapter PCB:
https://web.archive.org/web/20190125061335/https://github.com/sanni/cartreader/wiki/Flash--and-Eprom-Adapter
